src/components/Explore/utils.ts

```diff
@@ -149,12 +149,39 @@ function getDatasetIdByMetric(metric: ExploreMetric): DatasetId {
   }
 }
 
+export const formatDecimalAxis = (num: number, places = 2): string => {
+  if (num === null) {
+    return '-';
+  }
+  /**
+   * If value is between 100k and 1m, display in abbreviated form.
+   * Examples
+   * - 100,000 => 100k
+   * - 1,000,000 => 1.0m
+   * - 1,100,000 => 1.1m
+   */
```

Comment:
Thanks for adding examples!  If you have time you could add a few unit tests based on these examples (but feel free to punt)